app/scripts/controllers/centers/CreateCenterController.js

```diff
@@ -13,9 +14,12 @@
             resourceFactory.centerTemplateResource.get({staffInSelectedOfficeOnly:true},function (data) {
                 scope.offices = data.officeOptions;
                 scope.staffs = data.staffOptions;
-                scope.groups = data.groupMembersOptions;
                 scope.formData.officeId = data.officeOptions[0].id;
             });
+              resourceFactory.groupResource.getAllGroups(function (data) {
```

Comment:
Need to get groups which are under selected office Not suggestible to get all groups and then filter in UI